eval_test.go

```diff
@@ -174,7 +174,7 @@ func TestEvalActionNode(t *testing.T) {
 	RunJetTest(t, data, nil, "actionNode_Add3Minus", `{{ 2+1+4-3 }}`, fmt.Sprint(2+1+4-3))
 
 	RunJetTest(t, data, nil, "actionNode_AddIntString", `{{ 2+"1" }}`, "3")
-	RunJetTest(t, data, nil, "actionNode_AddStringInt", `{{ "1"+2 }}`, "12")
+	RunJetTest(t, data, nil, "actionNode_AddStringInt", `{{ "1"+2 }}`, "3")
```

Comment:
@tooolbox It does change the behavior of the + as can be seen by this test.

Reply:
Got it.  Yeah some other discussion on that in #32 

So yeah, I see the breaking change...but in thinking about it, how many people intended `"1"+2 = "12"`?

I will look for this scenario in my tests but now I doubt it's an issue.